atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/api/AtlasDbConstraintCheckingMode.java

```diff
@@ -16,25 +16,25 @@
 package com.palantir.atlasdb.transaction.api;
 
 public enum AtlasDbConstraintCheckingMode {
-    FULL_CONSTRAINT_CHECKING_THROWS_EXCEPTIONS(ShouldThrowException.YES),
-    FULL_CONSTRAINT_CHECKING_LOGS_EXCEPTIONS(ShouldThrowException.NO),
-    PERFORMANCE_OPTIMAL_CONSTRAINT_CHECKING_THROWS_EXCEPTIONS(ShouldThrowException.YES),
-    PERFORMANCE_OPTIMAL_CONSTRAINT_CHECKING_LOGS_EXCEPTIONS(ShouldThrowException.NO),
-    NO_CONSTRAINT_CHECKING(ShouldThrowException.NO);
+    FULL_CONSTRAINT_CHECKING_THROWS_EXCEPTIONS(ShouldThrow.YES),
+    FULL_CONSTRAINT_CHECKING_LOGS_EXCEPTIONS(ShouldThrow.NO),
+    PERFORMANCE_OPTIMAL_CONSTRAINT_CHECKING_THROWS_EXCEPTIONS(ShouldThrow.YES),
+    PERFORMANCE_OPTIMAL_CONSTRAINT_CHECKING_LOGS_EXCEPTIONS(ShouldThrow.NO),
+    NO_CONSTRAINT_CHECKING(ShouldThrow.NO);
 
-    private static enum ShouldThrowException {
+    private enum ShouldThrow {
         YES,
         NO
     }
 
-    private final ShouldThrowException shouldThrowException;
+    private final ShouldThrow shouldThrowException;
```

Comment:
Class name contained the word "exception" but didn't extend an `Exception` class. It's a rare case but a bit weird here.